app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -72,10 +73,8 @@ module.exports = function (defaults) {
         "node_modules/@discourse/backburner.js/dist/named-amd/backburner.js",
     },
 
-    historySupportMiddleware: false,
-
     trees: {
-      app: RawHandlebarsCompiler("app"),
+      app: RawHandlebarsCompiler(withSideWatch("app", { watching: ["../discourse-markdown-it"] })),
```

Comment:
```suggestion
    trees: {
      app: RawHandlebarsCompiler(
        withSideWatch("app", { watching: ["../discourse-markdown-it"] })
      ),
    },
```

This is what prettier wants (ran it locally). If you apply this CI should pass!